website/server/extractDocs.js

```diff
@@ -95,14 +125,12 @@ function filterMethods(method) {
 // Determines whether a component should have a link to a runnable example
 
 function isRunnable(componentName, componentPlatform) {
-  var path = '../Examples/UIExplorer/' + componentName + 'Example.js';
-  if (!fs.existsSync(path)) {
-    path = '../Examples/UIExplorer/' + componentName + 'Example.' + componentPlatform + '.js';
-    if (!fs.existsSync(path)) {
-      return false;
-    }
+  var paths = getExamplePaths(componentName, componentPlatform);
+  if (paths && paths.length > 0) {
```

Comment:
Actually `undefined && foo` evaluates to undefined so seems better to explicitly return `true`, `false` just to be safe.